### DISCUSSION TITLE
attach a layout to the inventory setup plugin

### DISCUSSION THREAD

**Discussion by greatfullbon** [runelite/runelite#18028] [Ideas]
I think it would be a good idea to add in the plugin "Bank Tags" a way to link our layout to the inventory setup plugin.

Probably by adding an option here when right clicking on the layout
![image](https://github.com/runelite/runelite/assets/7688988/3602f852-1e73-4ddb-a867-7d56a4e1b018)

So, when our invetory setup is open, it could automatically open our tag when we open our bank.

For my part, I make a layout for each inventory setup, which allows me to personalize my layouts.

here is an example
![image](https://github.com/runelite/runelite/assets/7688988/9c9a8974-f83c-4811-9de7-795224d62303)

it could be an option a bit like in the"Bank tag layout" pugin

![image](https://github.com/runelite/runelite/assets/7688988/12cb1abc-2552-49a8-a0da-68483253e49e)

Thank you !

**Comment by Nightfirecat** [comment]
This change would need to happen in the Inventory Setups plugin

**Reply by greatfullbon** [reply]
yes you are right

**Reply by Felanbird** [reply]
So you should open this suggestion on the [inventory setups](https://runelite.net/plugin-hub/show/inventory-setups) project

**Reply by greatfullbon** [reply]
how i do that? i post on their reddit or they have a github?

**Reply by Felanbird** [reply]
The Github link is available in the top-right corner of the README section on that page, it's just named `Report an Issue` on the RuneLite website because that's most commonly why people end up on the page.

**Reply by greatfullbon** [reply]
Thank you, it's done.
